### ISSUE TITLE
Pylint Issues

### ISSUE THREAD

**Issue by Shanmugapriya03** [Haider8/tmessage#34]
Currently, Pylint gives `Your code has been rated at 4.92/10`. This needs to be fixed to improve the quality of code and also we can add Pylint in CI so that new codes pushed in the future will have a check of Pylint 

To get the Pylint score: You can run `pylint *` from the terminal in the Project folder

**Comment by Haider8** [comment]
@Shanmugapriya03 This seems like a nice idea. Thanks for opening this issue. Would you like to work on this?

**Comment by Shanmugapriya03** [comment]
Yes @Haider8 assign it to me

**Comment by Shanmugapriya03** [comment]
@Haider8 I have fixed pylint errors in code from `4.92 to 7.52` Rating
Need your confirmation on fixing the below errors as it leads to removing some lines of code
```************* Module CODE_OF_CONDUCT
CODE_OF_CONDUCT.md:5:6: E0001: invalid syntax (<unknown>, line 5) (syntax-error)
************* Module CONTRIBUTING
CONTRIBUTING.md:3:8: E0001: invalid syntax (<unknown>, line 3) (syntax-error)
************* Module LICENSE
LICENSE:1:20: E0001: unexpected indent (<unknown>, line 1) (syntax-error)
************* Module README
README.md:5:1: E0001: invalid syntax (<unknown>, line 5) (syntax-error)
************* Module requirements
requirements.txt:1:15: E0001: invalid syntax (<unknown>, line 1) (syntax-error)
************* Module tmessage.cli
tmessage/cli.py:38:15: W0613: Unused argument 'client' (unused-argument)
tmessage/cli.py:38:23: W0613: Unused argument 'userdata' (unused-argument)
tmessage/cli.py:115:11: W0703: Catching too general exception Exception (broad-except)
tmessage/cli.py:108:8: W0104: Statement seems to have no effect (pointless-statement)
tmessage/cli.py:112:8: W0104: Statement seems to have no effect (pointless-statement)
tmessage/cli.py:116:8: W0104: Statement seems to have no effect (pointless-statement)
************* Module tmessage
tmessage.egg-info/__init__.py:1:0: F0010: error while code parsing: Unable to load file tmessage.egg-info/__init__.py:
[Errno 2] No such file or directory: 'tmessage.egg-info/__init__.py' (parse-error)
************* Module venv
venv/__init__.py:1:0: F0010: error while code parsing: Unable to load file venv/__init__.py:
[Errno 2] No such file or directory: 'venv/__init__.py' (parse-error)

------------------------------------------------------------------
Your code has been rated at 7.52/10 ```
